src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: #{$card-width};
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__subtitle {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__rating {
+    width: 100%;
+    margin-bottom: 24px;
+
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: #060b35;
```

```suggestion
      color: #060b35;
```
be aware of some repetitive codes that you can use variables instead especially color fields